checkout/checkout-risk-sdk-ios
# PRISM-11343 - Wrap fingerprint implementation within a timeout

## Issue

https://checkout.atlassian.net/browse/PRISM-11343

## Proposed changes

Wrap fingerprint implementation within a timeout

## Test Steps

> 1. On the `fingerprintTimeoutInterval`, change the `timeoutInterval` to 0.1.
> 2. Use an example app to run the application, it should log the timeout error
> 3. Revert the value after testing.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
